src/ClassicUO.Assets/ClassicUO.Assets.csproj

```diff
@@ -4,6 +4,14 @@
     <OutputType>Library</OutputType>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
```

Comment:
I think this prop can be made global putting it into:
https://github.com/ClassicUO/ClassicUO/blob/main/src/Directory.Build.props#L3-L7

Reply:
All done, same as below.